src/Symfony/Component/Validator/Constraints/CallbackValidator.php

```diff
@@ -44,8 +44,11 @@ public function validate($object, Constraint $constraint)
             }
 
             \call_user_func($method, $object, $this->context, $constraint->payload);
+        } elseif (\method_exists($method, '__invoke')) {
```

Comment:
Instead of adding this branch, previous one should be modified
```diff
- if ($method instanceof \Closure) {
+ if (is_callable($method)) {
```

Reply:
when we use ```\Closure``` and checking use ```is_callable()```, the result is always false in this case. Because ```$method``` is string contain the class name.

Reply:
You can check, when we use ```is_callable()``` we get an exception ```Method "%s" targeted by Callback constraint does not exist in class %s```

Reply:
Well it shouldn't be a string, my guess is you use it incorrectly. $method should be invokable, not a method of new instance of $method.

Reply:
@ostrolucky : i added a test to check if ```$method``` is object or not, can you check the new one?